ppx_cstruct.opam

```diff
@@ -23,5 +23,6 @@ depends: [
   "ocaml-migrate-parsetree"
   "ppx_driver"    {test & >= "v0.9.0"}
   "ppx_sexp_conv" {test}
+  "cstruct-unix"  {test}
```

Comment:
Also it's useful to have some of the matrix _not_ run tests, so you can verify that the package installs without test dependencies.